doc/history.html

```diff
@@ -134,6 +134,77 @@ <h3>Stable Series</h3>
         <ul>
           <!-- starting 3.6 generated by tags.yaml -->
           <!-- start stable -->
+          <li>
+            <a id="v3_10_9"><b>v3_10_9</b></a> released on January 15, 2025 (<a
```

Comment:
Is it okay if the date info says "released on January 15..." instead of January 16 (which is what CHANGELOG contains?